config/initializers/omniauth.rb

```diff
@@ -7,10 +7,19 @@ module Strategies
 end
 
 Rails.application.config.middleware.use OmniAuth::Builder do
+  options = { scope: "user:email,repo,delete_repo,admin:org,admin:org_hook" }
+
+  if GitHubClassroom.enterprise_instance?
+    hostname = Rails.application.secrets.github_enterprise_hostname
+    options[:site]          = "https://#{hostname}/api/v3"
+    options[:authorize_url] = "https://#{hostname}/login/oauth/authorize"
+    options[:token_url]     = "https://#{hostname}/login/oauth/access_token"
+  end
```

Comment:
These three properties appear under `client_options` now. (I already connected GitHub Classroom to GitHub Enterprise in our school.)

Reply:
Sorry, I'm not quite sure where you mean... do you mean in `github_client`? I can't find where these would be setup outside of this middleware.

Reply:
No, this is what I meant:
```ruby
options[:client_options] = {
  :site => "https://#{hostname}/api/v3",
  :authorize_url => "https://#{hostname}/login/oauth/authorize",
  :token_url => "https://#{hostname}/login/oauth/access_token",
}
```
See https://github.com/omniauth/omniauth-github.